ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -581,6 +581,40 @@ get_title_clause2 (EkncQueryObject *self, gchar **terms)
 }
 
 static gchar *
+get_title_clause3 (EkncQueryObject *self,
+                   char **terms,
+                   char **corrected_terms)
+{
+  guint terms_length = g_strv_length (terms);
```

Comment:
nitpick: you've changed `g`types to plain old types elsewhere, how about using `unsigned` or `size_t` here? (Same remark for `g_new0 (char *, ...)` a few lines below.)

Reply:
`gchar` is a ridiculous type, but `guint` saves characters. :-)

To be fair, I'd go over the whole thing and just use `bool` from `stdbool.h`, and the integer types from `stdint.h`, since we're using C99.